sdk/servicebus/Azure.Messaging.ServiceBus/src/Resources.resx

```diff
@@ -300,4 +300,7 @@
   <data name="OnlyOneSharedAccessAuthorizationMayBeSpecified" xml:space="preserve">
     <value>The authorization for a connection string may specifiy a shared key or precomputed shared access signature, but not both.  Please verify that your connection string does not have the `SharedAccessSignature` token if you are passing the `SharedKeyName` and `SharedKey`.</value>
   </data>
+  <data name="InvalidEndpointAddress" xml:space="preserve">
+    <value>The endpoint address could not be parsed; it was either malformed or not using the `sb://` scheme.</value>
```

Comment:
I thought not using the sb scheme was fine because we will add it in for you:
```
                       var endpointBuilder = new UriBuilder(value)
                        {
                            Scheme = ServiceBusEndpointScheme,
                            Port = -1
                        };
```

Reply:
The source can use any scheme so long as the builder can successfully replace it.  It is possible to mangle the value in a way that the scheme doesn't update.  This message corresponds to a post-builder check to be sure that the scheme formatting was successful.  It's some weird corner case that is covered by one of the parser test case sets.  _(I'm being lazy and not digging into the details, but we can discuss offline and find it, if you're interested.)_